### ISSUE TITLE
Insufficient contrast for $text-sub (a11y)

### ISSUE THREAD

**Issue by alexdahl-okta** [okta/odyssey#18]
The sub-text color is currently Gray 500 (`#899298`): https://github.com/okta/odyssey/blob/master/src/scss/abstracts/_colors.scss#L95

This color is used for text at small sizes, such as field hints.

<img width="517" alt="subtext" src="https://user-images.githubusercontent.com/26014482/59545005-09bab800-8eb4-11e9-8ce6-31ff99d1851c.png">

WCAG AA requires regular text to have a contrast ratio of at least 4.5:1: https://www.w3.org/TR/WCAG21/#contrast-minimum

In the system's current palette, the lightest gray with >4.5:1 contrast against white is Gray 700 (`#5c6971`).

**Comment by edburyenegren-okta** [comment]
This will be solved by https://github.com/okta/odyssey/pull/534
